flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/detail/job-checkpoints-detail.component.ts

```diff
@@ -80,5 +83,25 @@ export class JobCheckpointsDetailComponent implements OnInit {
       this.cdr.markForCheck();
       this.refresh();
     });
+    this.jobService.loadCheckpointConfig(this.jobDetail.jid).subscribe(config => {
+      this.checkPointConfig = config;
+      this.cdr.markForCheck();
+    });
+  }
+
+  checkPointType(){
```

Comment:
it would be better to checkPointType as a value other than a function here
then move the calculation to line 65

Reply:
done.